docker-compose-dev.yml

```diff
@@ -8,11 +8,11 @@ services:
       POSTGRES_USER: gondolin
       POSTGRES_PASSWORD: gondolin
     volumes:
-      - ./postgresql_data:/var/lib/postgresql/data:Z
+      - /var/data/gondolin-dev/postgresql_data:/var/lib/postgresql/data:Z
```

Comment:
Could we use a [named volume](https://docs.docker.com/storage/volumes/) here instead of an absolute path?